dolphinscheduler-ui/src/views/projects/task/components/node/fields/use-datax.ts

```diff
@@ -443,6 +444,7 @@ export function useDataX(model: { [field: string]: any }): IJsonItem[] {
       span: 12,
       options: memoryLimitOptions,
       value: 1
-    }
+    },
+    ...useCustomParams({ model, field: 'localParams', isSimple: false })
```

Comment:
This block should use simple `useCustomParams` components, the backend only uses key, value

```
...useCustomParams({ model, field: 'localParams', isSimple: true })
```